react/components/NotificationBar/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react'
+import { bool, string, oneOf } from 'prop-types'
+import classNames from 'classnames'
+
+import styles from './styles.css'
+
+const NotificationBar = ({
+  content
+}) => {
+
+  return content && (
+    <div className={`${styles.notificationBarContainer} bg-action-primary c-on-action-primary`}>
```

Comment:
perhaps a better default background would be `bg-base--inverted`, and likewise for the text color

Reply:
I will allow the user to choose between then.